batch/batch/driver/main.py

```diff
@@ -504,15 +503,17 @@ async def on_startup(app):
 
     machine_name_prefix = f'batch-worker-{DEFAULT_NAMESPACE}-'
 
-    credentials = google.oauth2.service_account.Credentials.from_service_account_file(
-        '/gsa-key/key.json')
-    gservices = GServices(machine_name_prefix, credentials)
-    app['gservices'] = gservices
-
+    aiogoogle_credentials = aiogoogle.Credentials.from_file('/gsa-key/key.json')
     compute_client = aiogoogle.ComputeClient(
-        PROJECT, credentials=aiogoogle.Credentials.from_file('/gsa-key/key.json'))
+        PROJECT, credentials=aiogoogle_credentials)
     app['compute_client'] = compute_client
 
+    logging_client = aiogoogle.LoggingClient(
+        credentials=aiogoogle_credentials,
+        # quota is 60/m, event loop runs 4/m
```

Comment:
Can you elaborate on this comment? If I understand this correctly, you create a client that has a rate limited session. The whole driver shares that session. How often the event loop fires shouldn't affect the rate limit, right? So why not set the `rate_limit` to `60, 60`?

Reply:
Yeah.  The project-wide quota is 60 request/m.  The event loop sleeps 15s per iteration, so the max rate is 4 iterations/m.  Note, the event loop could make multiple logging requests per iteration, so this numbers are not quite comparable.  I didn't want this to consume the entire quota since there will be other consumers of this quota (us at the web console, test deployments, etc.)

I can add this comment as a comment in the code if you want.

Reply:
Yes please